lib/kitchen/document.rb

```diff
@@ -165,7 +165,32 @@ def create_element_from_string(string)
     def record_id_copied(id)
       return if id.blank?
 
-      @next_paste_count_for_id[id] ||= 1
+      @id_cp_count[id][:count]&.positive? ? @id_cp_count[id][:count] += 1 : @id_cp_count[id][:count] = 1
```

Comment:
With the above change, and the fact that line 179 below prevents negative counts from ever happening, this reduces to 

```ruby
@id_cp_count[id][:count] += 1
```